ui/apps/dashboard/src/components/Onboarding/DeployApp.tsx

```diff
@@ -0,0 +1,247 @@
+import { useRouter } from 'next/navigation';
+import { NewButton } from '@inngest/components/Button';
+import CommandBlock from '@inngest/components/CodeBlock/CommandBlock';
+import { NewLink } from '@inngest/components/Link';
+import TabCards from '@inngest/components/TabCards/TabCards';
+import { IconCloudflare } from '@inngest/components/icons/platforms/Cloudflare';
+import { IconFlyIo } from '@inngest/components/icons/platforms/FlyIo';
+import { IconVercel } from '@inngest/components/icons/platforms/Vercel';
+import { RiCheckboxCircleFill, RiCloudLine } from '@remixicon/react';
+
+import { Secret } from '@/components/Secret';
+import { pathCreator } from '@/utils/urls';
+import { OnboardingSteps } from '../Onboarding/types';
+import useOnboardingStep from './useOnboardingStep';
+
+export default function DeployApp() {
+  const { updateLastCompletedStep } = useOnboardingStep();
+  const router = useRouter();
+
```

Comment:
This file will likely be split in components by platform for better organization, once I add the actions